### PR TITLE
ci: silence yamllint warnings

### PR DESCRIPTION
**Pull request by subhamkrai**
this commit silence yamllint warnings

Signed-off-by: subhamkrai <srai@redhat.com>

**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [ ] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.

### CODE REVIEW

File: .github/workflows/yaml-lint.yaml [hunk 1]
```diff
@@ -5,27 +5,20 @@ on:
       - "**.yaml"
 
 jobs:
-  yaml-linter:
+  test:
```
Comment (Madhu-1) [inline .github/workflows/yaml-lint.yaml:8]: still we are doping yaml-linter do we need to rename it

Comment (subhamkrai) [reply .github/workflows/yaml-lint.yaml]: done.

**Review by leseb — APPROVED** [review]

File: .github/workflows/yaml-lint.yaml [hunk 2]
```diff
@@ -8,24 +8,15 @@ jobs:
...
-              document-start: disable
-              comments: disable
-              truthy: disable
+          python-version: 3.8
+
+      - name: Install yamllint
+        run: pip install yamllint
+
+      - name: Lint YAML files
+        run: yamllint -c .github/workflows/.yamllint cluster/examples/kubernetes/ --no-warnings
```
Comment (leseb) [inline .github/workflows/yaml-lint.yaml:22]: Why did we stop using the action?

Comment (subhamkrai) [reply .github/workflows/yaml-lint.yaml:22]: the action was not command friendly to be specific `--no-warnings` was not working.

**Pull request merged**